cmd/end2endtest/zkweighted.go

```diff
@@ -17,13 +17,18 @@ func init() {
 		description: "Performs a complete test of anonymous election, from creating a census to voting and validating votes",
 		example:     os.Args[0] + " --operation=anonelection --votes=1000",
 	}
+	ops["anonelectionTempSIKS"] = operation{
```

Comment:
i'd rather keep the case consistent, TempSIKs instead of TempSIKS
(there are more ocurrences of this)

Reply:
added the suggestion :raised_hands: